package-overrides/github/components/jqueryui@1.11.3.json

```diff
@@ -1,7 +0,0 @@
-{
-    "main": "jquery-ui.js",
-    "registry": "jspm",
-    "dependencies": {
-        "jquery": "*"
-    }
-}
```

Comment:
""

It is better to leave old overrides in place I think, and if they are conflicts, then to create a newer override for newer versions that takes preference.